setenv.sh

```diff
@@ -31,6 +32,8 @@ export KDBTAIL=${TORQDATAHOME}/taildir
 # set rlwrap and qcon paths for use in torq.sh qcon flag functions
 export RLWRAP="rlwrap"
 export QCON="qcon"
+export QCMD="taskset -c 0,1 /usr/bin/rlwrap q"
+export QCMDOVERRIDE=true                                                  # override qcmd in processes.csv with QCMD defined here
```

Comment:
What does this do?

Reply:
I believe this makes it so you don't have to remove the q's from the process csv to run on homer but this was copied in from previous iterations of the setenv code.